### PR TITLE
d.dmkt-sp.jp

### PR DESCRIPTION
**Pull request by SKEIDs**
only anniversary banner 
 https://github.com/AdguardTeam/AdguardFilters/issues/101941

### CODE REVIEW

**Comment by zloyden** [comment]
I don't see any statistics for the website.

**Comment by SKEIDs** [comment]
> I don't see any statistics for the website.

sorry, Is this not a statistic?
https://www.similarweb.com/website/dmkt-sp.jp/?from_ext=1

**Comment by zloyden** [comment]
It's strange, I didn't have any visitor statistics before.

File: AnnoyancesFilter/sections/self-promo.txt [hunk 1]
```diff
@@ -50,6 +50,8 @@ disk.yandex.*##.mail360-banner
 !-----------------------------------------------
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/101941
+d.dmkt-sp.jp###Anniversary
```
Comment (zloyden) [inline AnnoyancesFilter/sections/self-promo.txt:54]: Have you reproduced it? I didn't see it on my end.

Comment (SKEIDs) [reply AnnoyancesFilter/sections/self-promo.txt:54]: It was there when I sent this pull request.
But now it has been replaced by a different banner.

`d.dmkt-sp.jp##.fltBnrBlock`

Comment (SKEIDs) [reply AnnoyancesFilter/sections/self-promo.txt:54]: > Have you reproduced it? I didn't see it on my end.

It was there when I sent this pull request. #102523
But now it has been replaced by another one. → `d.dmkt-sp.jp##.fltBnrBlock`
I have not reproduced it. → `d.dmkt-sp.jp###Anniversary`

Comment (zloyden) [reply AnnoyancesFilter/sections/self-promo.txt:54]: Well, remove `d.dmkt-sp.jp###Anniversary` and add `d.dmkt-sp.jp##.fltBnrBlock` to the `AdGuard Annoyances - popups`.

Comment (SKEIDs) [reply AnnoyancesFilter/sections/self-promo.txt]: ok.done

**Review by zloyden — APPROVED** [review]

**Review by Yuki2718 — APPROVED** [review]

**Pull request merged**